app/code/Magento/CatalogImportExport/Model/Import/Product/CategoryProcessor.php

```diff
@@ -108,7 +108,16 @@ protected function createCategory($name, $parentId)
         $category->setIsActive(true);
         $category->setIncludeInMenu(true);
         $category->setAttributeSetId($category->getDefaultAttributeSetId());
-        $category->save();
+        try{
+            $category->save();
+        }catch (\Magento\Framework\Exception\AlreadyExistsException $e){
+            // Category already exists, use it:
+            $urlPath = $category->getData('url_path');
+            $category = $this->categoryFactory->create()->loadByAttribute('url_path', $urlPath);
```

Comment:
@EliasKotlyar correct me if I'm wrong, this logic will use previously created category and silently override it with the data of the new category instead of throwing the exception. This can be dangerous: if there is mistake in the sku, categories will be overriden, while for user it is better to see the exception and correct the mistake.